src/features/authentication/components/LoginForm/Categories/Categories.css

```diff
@@ -0,0 +1,46 @@
+.sport-card {
```

Comment:
Please create a component for the CategoryThumbNail
should be in src/components/CategoryThumbNail
and inside CategoryThumbNail you should have:
CategoryThumbNail.jsx
CategoryThumbNail.styles.js
we are using styled components not css modules.